app/models/restitution/avec_entrainement/temps_entrainement.rb

```diff
@@ -3,10 +3,10 @@
 module Restitution
   class AvecEntrainement
     class TempsEntrainement < Restitution::Metriques::Base
-      def calcule
-        return nil if @evenements_entrainement.empty?
+      def calcule(_, evenements_entrainement)
```

Comment:
Après avoir lu la PR je me la question d'avoir des arguments nommés. On sait que l'on a un argument vide mais on ne sait jamais lequel à moins de lire le second argument.

Reply:
Comme ce code est beaucoup répété je m'attends a ce qu'on sache par cœur le contenu de ces deux paramètres et faire des paramètres nommés ici alourdirait beaucoup. Je préfères qu'on laisse comme ça.

Reply:
Bien noté ! C'est ok pour moi.